lib/sort-related-items.js

```diff
@@ -26,6 +26,18 @@ module.exports = (related, type) => {
   return {
     relatedObjects: sortedRelated.object,
     relatedDocuments: sortedRelated.archive,
-    relatedChildren: sortedRelated.children
+    relatedChildren: sortedRelated.children.sort((a, b) => {
+      var aIdentifier = a.attributes.identifier;
+      var bIdentifier = b.attributes.identifier;
+
+      if (aIdentifier < bIdentifier) {
+        return -1;
+      }
+      if (aIdentifier > bIdentifier) {
+        return 1;
+      }
+
```

Comment:
What is the type of identifier? Do you sort alphabetically? Are the identifier unique or do you need to add another condition where a.attributes.identifier === b.attributes.identifier?